src/perf-tests/gatling-scripts/Airline.scala

```diff
@@ -0,0 +1,197 @@
+                      .header("Authorization", "${token}" )
+                      .body(ElFileBody("queries/listBookings.json"))                 
+                      .asJson
+                      .check(bodyString.saveAs("Retrieve bookings"))
+                      .check(status.not(404), status.not(500)))
+                      .exec(session => {
+                        println(session)
+                        session
+                      })                    
+        .pause(1)
+  }
+
+object Make {
+
+      val randomString = csv(TOKEN_CSV).random
+
+      val headerMaps = Map("Content-Type" -> "application/x-www-form-urlencoded", 
+                              "Accept" -> "application/json",
+                              "Origin" -> "https://js.stripe.com",
+                              "User-Agent" -> "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36"                          
+                            ) 
+
+      val randSelect: Random = new Random()
+      val randNum: Int = randSelect.nextInt(50)         
+
+      val booking =  feed(randomString)
+                      .exec( _.set("YEAR", YEAR))
+                      .exec( _.set("DEPARTURE_CODE", DEPARTURE_CODE))
+                      .exec( _.set("ARRIVAL_CODE", ARRIVAL_CODE))    
+                      .exec(http("Search Rand Flights")
+                      .post(GRAPHQL_URL)
+                      .headers(headerMaps)
+                      .header("Authorization", "${token}" )
+                      .body(ElFileBody("queries/listFlights.json"))
+                      .asJson
+                      .check(bodyString.saveAs("Query Flights"))
+                      .check(jsonPath("$..id").find(randNum).saveAs("bookingId"))
+                      .check(jsonPath("$..ticketPrice").find(randNum).saveAs("ticketPrice"))
+                      .check(status.not(404), status.not(500)))
+                      .exec(session => {
+                        println(session)
+                        session
+                      })
+                      .exec(http("Stripe tokenization")
+                      .post("https://api.stripe.com/v1/tokens")
+                      .headers(headerMaps)
+                      .formParamMap(Map(
+                        "card[name]" -> "John",
+                        "card[address_zip]" -> "NC2+8234",
+                        "card[address_country]" -> "UK",
+                        "card[number]" -> "4242424242424242",
+                        "card[cvc]" -> "123",
+                        "card[exp_month]" -> "08",
+                        "card[exp_year]" -> "32",
+                        "guid" -> "7f94f22b-dd03-4d25-9482-48df472123dd",
+                        "muid" -> "999143d3-02bc-4fc3-bf5f-510270268507",
+                        "sid"  -> "5fa32dea-d7ca-4154-8efd-3181894f126f",
+                        "key"  -> "pk_test_BpxANYCOZO7waMV3TrPQHjXa"
```

Comment:
That's gonna cause issues as it'll be different than Stripe Public Key they setup.

Reply:
good catchup. Missed this one. Update the public key to be passed from Systems Manager as input to Gatling script